Resources/Prototypes/Voice/speech_emotes.yml

```diff
@@ -2,14 +2,18 @@
 - type: emote
   id: Scream
   category: Vocal
-  chatMessages: [screams, yells]
+  chatMessages: [screams!]
```

Comment:
mmmm, so this means /me screams won't scream? I think the actual issue here is that emotes need to have better loc strings

Reply:
it'll scream, it triggers on `screams` anyways